.github/workflows/snyk_sca_scan.yaml

```diff
@@ -15,17 +19,26 @@ jobs:
       matrix:
         node-version: [16.x]
     steps:
-      - uses: actions/checkout@v3
-      - name: Setting up Node
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          token: ${{ secrets.BOT_GITHUB_ACTION_TOKEN }}
+      - name: Set up Node.js
         uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node-version }}
+          node-version: 16
+          cache: 'yarn'
+      - name: Run yarn
+        run: yarn
+      - name: Run build
+        run: yarn build
       - name: Installing snyk-delta and dependencies
         run: npm i -g snyk-delta
       - uses: snyk/actions/setup@master
       - name: Perform SCA Scan
         continue-on-error: false
         run: |
-          snyk test --all-projects --strict-out-of-sync=false --detection-depth=6 --exclude=system-tests,docker,Dockerfile --severity-threshold=critical
+          snyk test --all-projects --strict-out-of-sync=false --detection-depth=6 --exclude=system-tests,tooling,cli,docker,Dockerfile --severity-threshold=critical
```

Comment:
@cypresschris What's the intention behind excluding the cli here?

Reply:
That CLI project was generating errors at scan runtime along with tooling. -- I thought these were both projects that could be excluded as auxiliary and not part of the release.

Reply:
@cypresschris The CLI has the dependencies that would be of most concern for our users. They are the dependencies that are included in the cypress cli on their system. I don’t think it should be excluded from scans.

Reply:
I've added `CLI` back @jennifer-shehane